modules/cnv_viewing_guest_ip_addresses.adoc

```diff
@@ -0,0 +1,59 @@
+[[viewing-guest-ip-addresses]]
+=== Viewing guest IP addresses
+
+You can view the IP addresses assigned to your Linux virtual machines by
+installing a QEMU guest agent on the host. When the guest agent is running,
+you can view the IP addresses per interface by checking the VMI status on
+the command line.
+
+.Procedure
+
+. Install the QEMU guest agent on the host:
++
+----
+$ yum install -y qemu-guest-agent
+----
+
+. Start the QEMU guest agent service:
++
+----
+$ systemctl start qemu-guest-agent
+----
+
+. View the IP address information for a VMI:
++
+----
+$ oc describe vmi <vmi_name>
+
+...
+Interfaces:
+   Interface Name:  eth0
+   Ip Address:      10.244.0.37/24
+   Ip Addresses:
+     10.244.0.37/24
+     fe80::858:aff:fef4:25/64
+   Mac:             0a:58:0a:f4:00:25
+   Name:            default
+   Interface Name:  v2
+   Ip Address:      1.1.1.7/24
+   Ip Addresses:
+     1.1.1.7/24
+     fe80::f4d9:70ff:fe13:9089/64
+   Mac:             f6:d9:70:13:90:89
+   Interface Name:  v1
+   Ip Address:      1.1.1.1/24
+   Ip Addresses:
+     1.1.1.1/24
+     1.1.1.2/24
+     1.1.1.4/24
+     2001:de7:0:f101::1/64
+     2001:db8:0:f101::1/64
+     fe80::1420:84ff:fe10:17aa/64
+   Mac:             16:20:84:10:17:aa
+----
+
+[NOTE]
```

Comment:
Is this note rendering where you want? I don't see this section in the preview, but I think you need a + on line 54.

Reply:
Good catch! I do want the note to be aligned with the final step. I'll add that with the style fixes. Thanks, Kathryn!